appveyor.yml

```diff
@@ -0,0 +1,44 @@
+# http://www.appveyor.com/docs/appveyor-yml
+
+# Don't actually build
+build: off
```

Comment:
What does this mean?

Reply:
I think this is a MSBuild thing. Node.js appveyor docs includes this line https://www.appveyor.com/docs/lang/nodejs-iojs/#quick-start